### PR TITLE
Add `service-workers race token` to request and its lookup

### PR DESCRIPTION
**Pull request by sisidovski**
This is a fetch spec update corresponding to the change in https://github.com/yoshisatoyanagisawa/ServiceWorker/pull/10

### CODE REVIEW

File: fetch.bs [hunk 1]
```diff
@@ -4523,6 +4527,22 @@ steps:
      <li><p>Return <var>fetchParams</var>'s <a for="fetch params">preloaded response candidate</a>.
     </ol>
 
+   <dt><var>request</var>'s <a for=request>service-workers race token</a> is not null
+   <dd>
+    <ol>
+     <li><p>Let <var>raceReesponse</var> to the result of [=lookup-race-response=] given <var>request</var>'s <a for=request>service-workers race token</a>, <var>request</var>'s <a for=request>reserved client</a>, <var>request</var>'s <a for=request>URL</a>.
```
Comment (yoshisatoyanagisawa) [inline fetch.bs]: Will [=lookup-race-response=] be defined in SW spec?

Comment (yoshisatoyanagisawa) [reply fetch.bs]: Also, I guess we only do the matching for HTTP/HTTPS and GET request.  Do we do such restriction here?

**Comment by yoshisatoyanagisawa** [comment]
This looks not a bikeshed code, I am not sure how external specifications can be linked here.

File: fetch.bs (hunk 1 above)
Comment (sisidovski) [reply fetch.bs]: > Will [=lookup-race-response=] be defined in SW spec?

Yes. It's added in https://github.com/yoshisatoyanagisawa/ServiceWorker/pull/10. I'm not sure if we have a good way to call an upcoming algorithm.

> Also, I guess we only do the matching for HTTP/HTTPS and GET request. Do we do such restriction here?

Those are described in https://github.com/yoshisatoyanagisawa/ServiceWorker/pull/10. The token is set under those restrictions. I guess we don't have to rephrase them.

Comment (yoshisatoyanagisawa) [reply fetch.bs]: Thank you.

**Review by yoshisatoyanagisawa — APPROVED** [review]
Then, I do not have any other comments.

**Comment by sisidovski** [comment]
Close this PR for now, and make another PR to the original fetch repository.

**Comment by sisidovski** [comment]
Now the PR in https://github.com/whatwg/fetch/pull/1737

**Pull request closed without merging**